Docs/source/usage/parameters.rst

```diff
@@ -2752,10 +2752,10 @@ The data collected at each boundary is written out to a subdirectory of the diag
 By default, all of the collected particle data is written out at the end of the simulation. Optionally, the ``<diag_name>.intervals`` parameter can be given to specify writing out the data more often.
 This can be important if a large number of particles are lost, avoiding filling up memory with the accumulated lost particle data.
 
-In addition to their usual attributes, the saved particles have an integer attribute ``step_scraped``, which
+In addition to their usual attributes, the saved particles have an integer attribute ``stepScraped``, which
 indicates the PIC iteration at which each particle was absorbed at the boundary,
-and a real attribute ``time_scraped``, which indicates the exact time calculated when each
-particle hits the EB.
+and a real attribute ``deltaTimeScraped``, which indicates the fraction of time between the time associated to the last step
+and the exact time when each particle hits the boundary. ``deltaTimeScraped`` is a real between 0 and dt.
```

Comment:
Thanks for adding this in the documentation.
Could you also add these attributes in the table here:
https://github.com/ECP-WarpX/WarpX/blob/development/Docs/source/developers/particles.rst#particle-attributes
In the `Notes`, you can add that this is only saved for particles collected at the boundaries (in the boundary buffers, and the corresponding `BoundaryScraping` Diagnostic)

Reply:
I opened a new PR for that:  #4719!